sdk/python/lib/pulumi_policy/policy.py

```diff
@@ -0,0 +1,707 @@
+# Copyright 2016-2020, Pulumi Corporation.
```

Comment:
Note: I started breaking up this file into smaller files, but ran into circular imports that I need to chase down, so keeping it all in one file for the time-being.